docker-berp/Dockerfile

```diff
@@ -0,0 +1,51 @@
+# Builds a docker image used for building most projects in this repo. It's
+# used both by contributors and CI.
+#
+FROM mcr.microsoft.com/dotnet/sdk:6.0
+
+ENV DEBIAN_FRONTEND=noninteractive
+
+RUN apt-get update \
+    && apt-get install --assume-yes \
+    locales
+
+RUN locale-gen en_US.UTF-8
+ENV LANG en_US.UTF-8
```

Comment:
Usually this is the only lang env variable needed for things like system locale.

i.e. when using ruby and doing `Time.now` this pulls in from this. Are the others needed for something else?

Reply:
The order of priority is:

1. LANGUAGE
2. LC_ALL
3. LC_xxx, while taking into account the locale category
4. LANG

And I ran into problems setting LANG. So I set the other two too. (I didn't know the order of priority then. Looked that up to answer your question...)